src/pages/Facility/settings/locations/LocationForm.tsx

```
@@ -263,7 +265,17 @@ export default function LocationForm({
   ];
 
   if (locationId && isLoading) {
-    return <div className="p-4">Loading...</div>;
+    return (
+      <div
+        className="p-4"
+        role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)